lib/schedule/gtfs/direction.ex

```diff
@@ -46,4 +46,16 @@ defmodule Schedule.Gtfs.Direction do
   @spec id_from_string(String.t()) :: id()
   def id_from_string("0"), do: 0
   def id_from_string("1"), do: 1
+
+  @doc """
+  Returns a inbound / outbound terms from a Direction ID
+
+  iex> Schedule.Gtfs.Direction.id_from_string(0)
+  "Outbound"
+  iex> Schedule.Gtfs.Direction.id_from_string(1)
+  "Inbound"
+  """
+  @spec id_to_string(id()) :: String.t()
+  def id_to_string(0), do: "Outbound"
+  def id_to_string(1), do: "Inbound"
```

Comment:
Gtfs does not specify if 0 means outbound and 1 means inbound, and we have a `direction_names` map on our route objects, are we certain that this will _always_ be the case?

Reply:
Did some digging, it seems like at the MBTA we should be using values from [`directions.txt`](https://github.com/mbta/gtfs-documentation/blob/master/reference/gtfs.md#directionstxt)?

Reply:
Whooaaaa I did NOT realize that GTFS doesn't specify this?! I received that by word of mouth over time, which means a lot of folks might be making the same assumption? 😳  Ok, I'll use `direction_names`. Thanks for the pointer!!